### PR TITLE
hw #6

### PR DESCRIPTION
**Pull request by IgorMuchnik**

### CODE REVIEW

File: src/test/java/hw6/steps/GivenSteps.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+  @Given("I navigate to Different Elements page through \"Service\" button in Header")
+  public void iOpenDifferentElementsPageThroughHeaderOnIndexPage() {
+    indexPage.navigateToDifferentElementsPage();
+  }
+
+  @Given("I navigate to User Table page through \"Service\" button in Header")
+  public void iOpenUserTablePageThroughHeaderOnIndexPage() {
+    indexPage.navigateToUserTablePage();
+  }
```
Comment (DmitryKhodakovsky) [inline src/test/java/hw6/steps/GivenSteps.java:31]: Looks like code duplication. What is the purpose of these two steps?

File: src/test/java/hw6/steps/ThenSteps.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+      entry.get("user"),
+      entry.get("description"));
+  }
+
+  @Then("User table should contain following values:")
+  public void iAssertContentOfTableOnUserTablePAge(List<JDIUser> users) {
+    List<JDIUser> actualUserTable = new ArrayList<>();
+    List<String> numbers = userTablePage.getUsersIds();
+    List<String> usernames = userTablePage.getUsersNames();
+    List<String> description = userTablePage.getUsersImagesDescriptions();
```
Comment (DmitryKhodakovsky) [inline src/test/java/hw6/steps/ThenSteps.java:82]: Why do you do not receive `JDIUser` from the page object?

**Review by DmitryKhodakovsky — APPROVED** [review]
Approved with comment

File: src/test/java/hw6/steps/GivenSteps.java (hunk 1 above)
Comment (IgorMuchnik) [reply src/test/java/hw6/steps/GivenSteps.java:31]: These steps navigate to the _different_ pages. But I agree to it should be optimized by one method with a pass of curtain item from the header.

**Pull request merged**